lib/bundler/settings.rb

```diff
@@ -35,6 +35,7 @@ class Settings
       frozen
       gem.coc
       gem.mit
+      github_https
```

Comment:
This is the list of settings, not feature flags, so the `.` should be used here, right?